packages/core/package.json

```diff
@@ -24,7 +24,10 @@
   "types": "./types/index.d.ts",
```

Comment:
the original "types" remains here for backwards compatibility reasons with older TS